charts/prometheus/templates/sts.yaml

```
@@ -101,6 +101,10 @@ spec:
             - name: config-volume
               mountPath: /etc/config
               readOnly: true
+          {{- with .Values.configmapReload.prometheus.extraVolumeMounts }}
+              extraVolumeMounts:
+                {{ toYaml . | nindent 16 }}
+          {{- end }}
```

- Indentation goes too far
- `extraVolumeMounts` is just a values' field, it cannot be included in the spec
- A blank line gets inserted

```suggestion
          {{- with .Values.configmapReload.prometheus.extraVolumeMounts }}
             {{- toYaml . | nindent 12 }}
          {{- end }}
```

---

Please checkout the latest commit.